packages/settingregistry/src/plugin-schema.json

```diff
@@ -102,6 +102,10 @@
     "jupyter.lab.transform": {
       "type": "boolean",
       "default": false
+    },
+    "version": {
+      "description": "The effective version of this schema",
```

Comment:
For information, what do you expect as `version` specifier? A version number, the schema id URI,... ?

Reply:
Ah, that may be an error: this file itself has a `version`, which i believe was running afoul of the initial `strict` mode, which i eventually disabled everywhere, as it goes beyond (any of) the metaschemas.

I'll fix it up.